pybamm/parameters/standard_parameters_lithium_ion.py

```diff
@@ -1,29 +1,18 @@
 #
-# Standard parameters for battery models
+# Standard parameters for lithium-ion battery models
 #
 """
 Standard pybamm.Parameters for battery models
 
 Physical Constants
 ------------------
-R
+sp.R
     Ideal gas constant
 sp.F
     Faraday's constant
-T
+sp.T
```

Comment:
should we call this T_ref in anticipation of including thermal effects?

Reply:
yeah I think that is sensible.